styles.css

```diff
@@ -0,0 +1,100 @@
+body {
+  background-image: url(/images/Portfolio\ background\ shapes\ mobile.png);
+  background-size: cover;
+  background-color: #191765;
+  color: #f5f5ff;
+  box-sizing: border-box;
+  font-family: sans-serif, Roboto;
+  font-style: normal;
+}
```

Comment:
- [ ] Consider making this image look like the one in your design template so as to match your design template 😄.

Your design:
<img width="230" alt="Loom_xSZ9pyb3Ei" src="https://user-images.githubusercontent.com/81468675/148375971-5548fe8a-bcd4-4450-aa0e-3a4ad8a3d57d.png">

Figma design:
<img width="180" alt="Loom_DowJn70XLY" src="https://user-images.githubusercontent.com/81468675/148376011-240b4720-fee1-4104-9a95-3e7f372d02c4.png">